src/webpack.config.externals.ts

```diff
@@ -85,20 +86,20 @@ module.exports = (env :Environment = {}) => {
 
   let EXTERNALS = EXTERNALS_DEFAULT;
   //console.debug('EXTERNALS', toStr(EXTERNALS));
-  const FILE_PATH_ABSOLUTE_R4X_CONFIG_JSON = join(DIR_PATH_ABSOLUTE_PROJECT, FILE_NAME_R4X_CONFIG_JSON);
+  const FILE_PATH_ABSOLUTE_R4X_CONFIG_JS = join(DIR_PATH_ABSOLUTE_PROJECT, FILE_NAME_R4X_CONFIG_JS);
   try {
-    const configJsonStats = statSync(FILE_PATH_ABSOLUTE_R4X_CONFIG_JSON);
+    const configJsonStats = statSync(FILE_PATH_ABSOLUTE_R4X_CONFIG_JS);
     if (configJsonStats.isFile()) {
-      const config = require(FILE_PATH_ABSOLUTE_R4X_CONFIG_JSON);
-      //console.debug('config', config);
+      const config = require(FILE_PATH_ABSOLUTE_R4X_CONFIG_JS);
+      //console.debug('config', toStr(config));
       if (config.externals) {
         EXTERNALS = Object.assign(config.externals, EXTERNALS);
       }
-    } // if FILE_NAME_R4X_CONFIG_JSON
+    } // if FILE_NAME_R4X_CONFIG_JS
```

Comment:
Personally, I think it's so easy to find matching squigglybraces, that adding a finalizing comment is excessive and noisy, could we remove them from at least short blocks like this?